### ISSUE TITLE
A fundamental parameter is missing

### ISSUE THREAD

**Issue by bwanaaa** [orlandott/Predictive_Policing#1]
It from your model, but from ALL deep learning models. The concept of a confidence interval/error bar. Or more accurately - a way to quantify the fallibility of using a population statistic to infer a characteristic of an individual  event. Situations where this is significant are: predicting the next event in a time series, predicting a value in a geographical dataset, predicting motion of a particle in a population. Other situations are different. Some problems are better suited to AI because the thing you are predicting (whether something is a cat) is causally related to the variables being measured in a very direct way.

But the problem remains, is it fair to use a population statistic to model an individual event probability? In physics we speak of the Heisenberg uncertainty principle when we try to predict the location and momentum of a particle concurrently. Though the energy/mass/velocity of the population of particles is measurable, the knowing of these values for a given individual carry uncertainty.

Deep learning models certainly have a ‘loss’function and a predictive accuracy statistic, but this is based on two separate populations - the training set and the testing set. These statistics are there fore only population statistics and valid. But how does one go from a population statistic to a number that is applicable to an individual event? For example, can a deep learning model be used to say whether someone committed a crime, whether a stock price will reach a certain number at a certain time. If we use linear regression, sometimes people will use the correlation coefficient as a measure of probability to estimate the likelihood of an additional event satisfying the correlation. There is no idea of the ‘error bar’ with a deep learning model. One possible way that might be interesting is to see how the the accuracy of a deep learning model changes with the addition of more hidden layers. That is a crude example but there are many other ways- changing the values in of the loss function, changing the optimization methods(gan, Adam,etc)

So the interesting idea your blog presents is the use of different deep learning models to analyze the same data. In other words, when we try to describe a population many different ways, we generTe a population of models. The collection of these models can then be described using a population statistic. If these models converge/or are ‘close to each other in success rate’, can we infer that the data are therefore ‘model-able’? Or is there a suite of models on which we can standardize so we can generate a ‘validity’ statistic?
